sync/sync.go

```diff
@@ -268,6 +305,14 @@ func MakeBranchPRs(f *Flags, dir string, entry *ConfigEntry) ([]SyncResult, erro
 		return nil, err
 	}
 
+	var reviewAuther gitcmd.URLAuther
+
+	if f.GitHubPATReviewer != nil {
+		reviewAuther = &gitcmd.GitHubPATAuther{
+			PAT: *f.GitHubPATReviewer,
+		}
+	}
```

Comment:
Possible `nil` use?

Reply:
Good spot, guessing I can fix this by initialising `reviewAuther` like this:

```go
var reviewAuther gitcmd.URLAuther = gitcmd.NoAuther{}
```